ext/gmp/gmp.stub.php

```diff
@@ -6,105 +6,105 @@ class GMP
 {
 }
 
-function gmp_init(int|string $number, int $base = 0): GMP {}
+function gmp_init(int|string $num, int $base = 0): GMP {}
 
-function gmp_import(string $data, int $word_size = 1, int $options = GMP_MSW_FIRST | GMP_NATIVE_ENDIAN): GMP {}
+function gmp_import(string $data, int $word_size = 1, int $flags = GMP_MSW_FIRST | GMP_NATIVE_ENDIAN): GMP {}
 
-function gmp_export(GMP|int|string $gmpnumber, int $word_size = 1, int $options = GMP_MSW_FIRST | GMP_NATIVE_ENDIAN): string {}
+function gmp_export(GMP|int|string $num, int $word_size = 1, int $flags = GMP_MSW_FIRST | GMP_NATIVE_ENDIAN): string {}
 
-function gmp_intval(GMP|int|string $gmpnumber): int {}
+function gmp_intval(GMP|int|string $num): int {}
 
-function gmp_strval(GMP|int|string $gmpnumber, int $base = 10): string {}
+function gmp_strval(GMP|int|string $num, int $base = 10): string {}
 
-function gmp_add(GMP|int|string $a, GMP|int|string $b): GMP {}
+function gmp_add(GMP|int|string $num1, GMP|int|string $num2): GMP {}
 
-function gmp_sub(GMP|int|string $a, GMP|int|string $b): GMP {}
+function gmp_sub(GMP|int|string $num1, GMP|int|string $num2): GMP {}
 
-function gmp_mul(GMP|int|string $a, GMP|int|string $b): GMP {}
+function gmp_mul(GMP|int|string $num1, GMP|int|string $num2): GMP {}
 
-function gmp_div_qr(GMP|int|string $a, GMP|int|string $b, int $round = GMP_ROUND_ZERO): array {}
+function gmp_div_qr(GMP|int|string $num1, GMP|int|string $num2, int $round = GMP_ROUND_ZERO): array {}
 
-function gmp_div_q(GMP|int|string $a, GMP|int|string $b, int $round = GMP_ROUND_ZERO): GMP {}
+function gmp_div_q(GMP|int|string $num1, GMP|int|string $num2, int $round = GMP_ROUND_ZERO): GMP {}
 
-function gmp_div_r(GMP|int|string $a, GMP|int|string $b, int $round = GMP_ROUND_ZERO): GMP {}
+function gmp_div_r(GMP|int|string $num1, GMP|int|string $num2, int $round = GMP_ROUND_ZERO): GMP {}
 
 /** @alias gmp_div_q */
-function gmp_div(GMP|int|string $a, GMP|int|string $b, int $round = GMP_ROUND_ZERO): GMP {}
+function gmp_div(GMP|int|string $num1, GMP|int|string $num2, int $round = GMP_ROUND_ZERO): GMP {}
 
-function gmp_mod(GMP|int|string $a, GMP|int|string $b): GMP {}
+function gmp_mod(GMP|int|string $num1, GMP|int|string $num2): GMP {}
```

Comment:
```suggestion
function gmp_mod(GMP|int|string $num, GMP|int|string $mod): GMP {}
```
To align with BCMath?